src/content/docs/dropins/user-auth/containers/sign-up.mdx

```diff
@@ -117,7 +105,7 @@ The SignUp container provides the following configuration options:
       'slots',
       'function',
       'No',
-      'Allows passing the SuccessNotification container or custom component rendered on successful sign-up if routeRedirectOnSignIn is not provided.',
+      'Allows passing the SuccessNotification or PrivacyPolicyConsent slot, or other custom component rendered upon successful sign-up, if routeRedirectOnSignIn is not provided.',
```

Comment:
This one might be a bit unclear. 

We have a SuccessNotification **slot**, which allows passing a SuccessNotification **container** or any other custom component to be rendered upon successful sign-up, if routeRedirectOnSignIn is not provided. Additionally, we have a PrivacyPolicyConsent slot, which allows passing a custom component to display a static privacy consent message in the sign-up form.